ros2/kachaka_grpc_ros2_bridge/src/dynamic_tf_bridge.cpp

```diff
@@ -49,8 +50,9 @@ bool ConvertGrpcTfToRosTf(
 namespace kachaka::grpc_ros2_bridge {
 
 TfStreamClient::TfStreamClient(
-    std::shared_ptr<kachaka_api::KachakaApi::Stub> stub, rclcpp::Node* node)
-    : stub_(stub),
+    std::string frame_prefix, std::shared_ptr<kachaka_api::KachakaApi::Stub> stub, rclcpp::Node* node)
+    : frame_prefix_(frame_prefix),
```

Comment:
ここでは move した方が良いですね。
```suggestion
    : frame_prefix_(std::move(frame_prefix)),
```

Reply:
コメントありがとうございました．
確かに，メモリの管理上では`std::move`を使用した方が適切だと考えます．